src/Python/wcon/wcon_schema.json

```diff
@@ -41,23 +41,13 @@
 			"description": "Information about experimental conditions and software versions; critical for reproducibility across labs.",
 			"properties": {
 				"lab": {"type": "object"},
-				"who": {"type": "string"},
+				"who": {"oneOf": [
```

Comment:
Why does this file exist?  Shouldn't everyone just use the root-directory wcon_schema.json?